### DISCUSSION TITLE
Sharepoint PNP module is getting auto updated to  newer versions

### DISCUSSION THREAD

**Discussion by Ramyaaarav** [pnp/powershell#1977] [Ideas]
Hi All,

We have robotic process automaiton(RPA) uses cases which uses sharepoint pnp modue to launch sharepoint sites. we use the version 3.10.1906.0 of SharePointPnPPowerShellOnline which is tested and used by our bots. BUt recently observing the version is getting auto updated in azure windows virtual machines....

**Comment by AndersRask** [comment]
Hey!

Be aware that this github project is PnP.PowerShell, which is not the same as SharePointPnPPowerShellOnline.
SharePointPnPPowerShellOnline has been deprecated, and should only be used for SharePoint on-premises.

If you have questions to this deprecated module you could try and ask them here
https://github.com/pnp/PnP-PowerShell
